scripts/gulpfiles/package_tasks.js

```diff
@@ -391,6 +392,8 @@ function packageDTS() {
     'typings/msg/msg.d.ts',
   ];
   return gulp.src(handwrittenSrcs, {base: 'typings'})
+      .pipe(gulp.src(`${BUILD_DIR}/ts/**/*.d.ts`))
```

Comment:
Please use `TYPINGS_BUILD_DIR` from `scripts/gulpfiles/config.js`—or even better—read the path from `tsconfig.json`.

(There are a few other paths that we should read from there too.)

Failing that, at minimum use the same path that `tsconfig.json` does (i.e. in this case):

```suggestion
      .pipe(gulp.src(`${BUILD_DIR}/ts/declarations/**/*.d.ts`))
```

and delete the TYPINGS_BUILD_DIR declaration, which appears to be unused.